create-cloud-resources.yml

```diff
@@ -51,6 +51,7 @@
           security_groups:
             - sg-common
             - sg-http-https
+            - sg-logstash
```

Comment:
This just associates the SG to the instance, you need to actually create the sg-logstash group and rule at https://github.com/BonnyCI/hoist/blob/master/roles/cloud-bootstrap/defaults/main.yml#L75